imager/imager_images/admin.py

```diff
@@ -0,0 +1,68 @@
+from django.contrib import admin
+from imager_images.models import Photo, Album
+import gettext
+
+
+
+class SizeListFilter(admin.SimpleListFilter):
+    title = gettext.gettext('File size')
+
+    parameter_name = 'size_category'
+
+    def lookups(self, request, model_admin):
+        return (
+                ('<=1MB', gettext.gettext('<= 1MB')),
+                ('<=10MB', gettext.gettext('<= 10 MB')),
+                ('<=100MB', gettext.gettext('<= 100 MB')),
+                ('>100MB', gettext.gettext('> 100 MB'))
+            )
+    def queryset(self, request, queryset):
+
+        if self.value() == '<=1MB':
+            return queryset.filter(picture__size=(0, 1000))
+        if self.value() == '<=10MB':
+            return queryset.filter(picture__size=(0, 10000))
+        if self.value() == '<=100MB':
+            return queryset.filter(picture__size=(0, 100000))
+        else:
+            return queryset.filter(picture__size=(100000, float('inf')))
```

Comment:
Be a bit cautious here.  Filters can be applied even when no value is present.  You should probably save the default for no filter at all and then implement this as a last option tied to the specific value `>100MB`